charts/karavi-observability/values.yaml

```diff
@@ -78,12 +78,43 @@ karaviMetricsPowerstore:
     serviceName: metrics-powerstore
     probability: 0.0
 
+karaviMetricsPowerscale:
+  image: dellemc/csm-metrics-powerscale:nightly
+  enabled: true
+  collectorAddr: otel-collector:55680
+  # comma separated list of provisioner names (ex: csi-isilon.dellemc.com)
+  provisionerNames: csi-isilon.dellemc.com
+  # set polling frequency to the PowerScale cluster to get metrics data
+  volumePollFrequencySeconds: 20
+  clusterPollFrequencySeconds: 300
```

Comment:
@taohe1012 , `volumePollFrequencySeconds` and `clusterPollFrequencySeconds` should be basically the same. `30` may be good value for them.

Reply:
@baoy1 update cluster interval to 30, and keep volume interval 20